docs/getting-started/setup.mdx

```diff
@@ -1,440 +1,130 @@
-        ```
-    </CodeGroup>
-
-
-2. Open http://localhost:6789 in your browser and build a pipeline.
-
-3. Run pipeline after building it in the tool
-
-   ```bash
-   docker run -it -p 6789:6789 -v $(pwd):/home/src mageai/mageai \
-     /app/run_app.sh mage run [project_name] [pipeline]
-   ```
-
-### Initialize new project
-
-If you want to create a different project with a different name, run the
-following:
-
-```bash
-docker run -it -p 6789:6789 -v $(pwd):/home/src mageai/mageai \
-  mage init [project_name]
-```
-
-### Connect to an external database
-
-<Note>
-
-Prior to Mage version 0.9.12, you may run into issues using database connection urls with special characters
-in the password such as "%" or "@" symbols. Please upgrade to a Mage version >= 0.9.12 to resolve these issues.
+## 🪄 Get Mage
 
-</Note>
+<AccordionGroup>
+   <Accordion title="Docker compose template" icon="Docker">
+   ### Get started
 
-Mage uses SQLite as the default database engine, but SQLite is not recommended for production use cases.
-The following databases are other alternatives that you can use with Mage.
+   The following is useful if you plan on customizing your image or adding additional services to a Docker network with Mage. Read more about Docker compose [here](https://docs.docker.com/compose/).
 
-#### PostgreSQL
-
-To use PostgreSQL as the database engine, you'll need to run your Docker container slightly differently:
-
-1. Create Docker network
-
-   ```bash
-   docker network create mage-app
-   ```
-
-2. Start PostgreSQL Docker container in a separate window
-
-   ```bash
-   docker run --network mage-app --network-alias postgres_db \
-      -it -p 5432:5432 -v pgdata:/var/lib/postgresql/data \
-      -e POSTGRES_USER=<username> -e POSTGRES_PASSWORD=<password> \
-      -e POSTGRES_DB=<database> -e PG_DATA=/var/lib/postgresql/data/pgdata \
-      postgres:13-alpine3.17 postgres
-   ```
-
-3. Launch Mage with `MAGE_DATABASE_CONNECTION_URL` environment variable
-
-   ```bash
-   docker run --network mage-app -it -p 6789:6789 -v $(pwd):/home/src \
-      -e MAGE_DATABASE_CONNECTION_URL=postgresql+psycopg2://<username>:<password>@postgres_db:5432/<database> \
-      mageai/mageai \
-      /app/run_app.sh mage start [project_name]
-   ```
-
-To use a specific schema in the PostgreSQL database, the connection string could contain the schema name as a parameter:
+   Be sure that [Docker](https://docs.docker.com/engine/install/) and [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) are installed and run:
 
    ```bash
-   docker run --network mage-app -it -p 6789:6789 -v $(pwd):/home/src \
-      -e MAGE_DATABASE_CONNECTION_URL="postgresql+psycopg2://<username>:<password>@postgres_db:5432/<database>?options=-c%%20search_path%%3D<schema_name>" \
-      mageai/mageai \
-      /app/run_app.sh mage start [project_name]
+   git clone https://github.com/mage-ai/compose-quickstart.git mage-quickstart \
```

Comment:
One thing I'm not sure is whether it works on windows.

Reply:
Those are all pretty universal commands (git clone, cd, cp, rm) so it should work perfectly.

Reply:
Some of these commands on different on windows: https://www.geeksforgeeks.org/linux-vs-windows-commands/

Reply:
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/copy-item?view=powershell-7.3#notes

`cp` is aliased to `Copy-Item`

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/remove-item?view=powershell-7.3#notes

`rm` is aliased to `Remove-Item`

`cd` is the same

Powershell does not use `&&`

```
git clone https://github.com/mage-ai/compose-quickstart.git mage-quickstart
   cd mage-quickstart
   cp dev.env .env
   rm dev.env
   docker compose up
```
updated